app/views/guests/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= "Guest Name: #{@guest.name}" %></h1>
+
+<% @guest_rooms.each do |room| %>
+  <div class="room" id="room-<%= room.id %>">
+    <p><%= "Hotel: #{room.hotel.name}" %></p>
+    <p><%= "Suite: #{room.suite}" %></p>
+    <p><%= "Rate: #{room.rate}" %></p>
+  </div>
+<% end %>
+
+<div class="room" id="new_room">
+  <h3>Add New Room</h3>
+  <%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |form| %>
```

Comment:
I like the RESTful URL + method here 👍

Reply:
The other option would be to do `url: /room_guests` and then have a hidden field in your form for `guest_id`